boards/neo_trinkey/src/lib.rs

```diff
@@ -0,0 +1,62 @@
+#![no_std]
+
+#[cfg(feature = "rt")]
+pub use cortex_m_rt::entry;
+
+pub use atsamd_hal as hal;
+pub use embedded_hal as ehal;
+pub use hal::pac;
+
+#[cfg(feature = "usb")]
+pub use hal::usb::{usb_device::bus::UsbBusAllocator, UsbBus};
+
+hal::bsp_pins!(
+    PA03 {
+        name: touch1,
+        aliases: {
+            AlternateB: Touch1
+        }
+    },
+
+    PA05 {
```

Comment:
```suggestion
    PA15 {
```

ref: https://github.com/adafruit/Adafruit-NeoKey-Trinkey-PCB/blob/main/Adafruit%20NeoKey%20Trinkey%20Pinout.pdf